### DISCUSSION TITLE
Welcome to GraphScope Discussions!

### DISCUSSION THREAD

**Discussion by yecol** [alibaba/GraphScope#30] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MultiChen** [comment]
![graphscope_session](https://user-images.githubusercontent.com/46548409/106245708-df17a180-6247-11eb-8c46-b2cfd892ca54.png)
Has anyone ever encountered this error?

**Comment by brahimi-m** [comment]
Édité photo
